static/src/javascripts/projects/commercial/modules/dfp/Advert.js

```diff
@@ -16,7 +16,7 @@ type Resolver = (x: boolean) => void;
  * One size       - `data-mobile="300,50"`
  * Multiple sizes - `data-mobile="300,50|320,50"`
  */
-const createSizeMapping = (attr: string): Array<AdSize> =>
+const createSizeMapping = (attr: string): Array<SingleSize> =>
```

Comment:
Yes, absolutely. I merely copied the definitions from the GPT reference but they are off putting. I see synergies with this file and the `ad-sizes` module, can we try to reconcile them? Maybe by updating `AdSize` this way:
```
type GuAdSize = SingleSizeArray* & { switchUnitId: ?number, toString: (_: void) => string }
```
NB: one could also leverage Flow support for literal types
```
type MPUSwitchId = 228;
type LeaderboardSwitchId = 229;
type BillboardSwitchId = 229;
type SwitchUnitId = MPUSwitchId | LeaderboardSwitchId | BillboardSwitchId;
```
and update the signature of `getAdSize` accordingly.

What say you?